collect_app/src/main/java/org/odk/collect/android/activities/GoogleSheetsUploaderActivity.java

```diff
@@ -555,8 +567,10 @@ public void authRequest(Uri url, HashMap<String, String> doneSoFar) {
                 uploadInstances(selection, selectionArgs, token);
             } catch (UserRecoverableAuthException e) {
                 mResults = null;
+                mAuthFailed = true;
```

Comment:
This branch is the branch that lets the user give permissions to their drive. It's not the same as the others and `mAuthFailed` shouldn't be modified here.